Update detailed guidance with new draft (#304)
DFE-Digital/get-into-teaching-content: content/guidance/become-a-teacher-in-england.md

### Teaching under fives

> If you choose to teach under fives, you’ll need to apply for a specialist teacher training course called Early Years Initial Teacher

Reviewer: missing full stop?

This is one of the most important parts of your application, so it’s worth thinking about it in advance.

You should use this section to showcase your motivation, commitment and teaching potential, backing up your answer with specific examples.

Give providers an insight into your personality by writing honestly and thoughtfully. Avoid cliché and write in clear, correct, concise English.

To get help with personal statements from a teacher training adviser, [use the Get an adviser service](/tta-service).

You don't have to cover everything in this list, but suggested topics
include:

  - why you want to be a teacher
  - your passion for your subject and the age group you’ve chosen to teach
  - the welfare and education of children and/or young people
  - the demands and rewards of the profession
  - personal qualities that will make you a good teacher
  - your contribution to the life of the school outside the classroom –
    for example, running extra-curricular activities and clubs
  - if you have school experience or have worked as a volunteer with children or young people, give details of what this has taught you

#### What do you know about the subject you want to teach?

> The application form also asks for detailed evidence for the knowledge and interest you bring to the subject(s) you’d like to teach.

Reviewer: Usually following GDS guidance we'd just use the plural here rather than the brackets around the 's'

> Do not use round brackets to refer to something that could either be singular or plural, like ‘Check which document(s) you need to send to DVLA.’

I know this was here prior to your change but it's probably worth tidying up as we go.

### Attend an interview

If your application is successful, you’ll be invited for an interview by your training provider.

> Interviews usually last a day and include an interview and some time spent teaching a class of real pupils.

Reviewer: This sounds a touch clunky, I'd be worried if the interview _didn't_ include an interview.

Per
```suggestion
Interviews usually last a day and may include some time spent teaching a class of real pupils.
```
